tests/unit/test_strike_service.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+from pytest_mock.plugin import MockerFixture
+from moddingway.database.models import Strike
+from moddingway.services import strike_service
+from moddingway import enums
+from typing import List
+import discord
+
+
+@pytest.mark.asyncio
```

Comment:
Are you having issues running async tests without this annotation? I think they should work without it, but if you have issues I'll make sure to add this to any async tests that I write as well

Reply:
It skips the tests when I dont have this, tells me to install plugins I already have:
 PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
  You need to install a suitable plugin for your async framework, for example:
    - anyio
    - pytest-asyncio
    - pytest-tornasync
    - pytest-trio
    - pytest-twisted
    warnings.warn(PytestUnhandledCoroutineWarning(msg.format(nodeid)))

Reply:
It skips the tests when I dont have this, tells me to install plugins I already have:
PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
You need to install a suitable plugin for your async framework, for example:
- anyio
- pytest-asyncio
- pytest-tornasync
- pytest-trio
- pytest-twisted
warnings.warn(PytestUnhandledCoroutineWarning(msg.format(nodeid)))

Reply:
I have the pipeline configured to install `pytest-asyncio` if you want, but I think the annotation works fine and I'll try to make sure to include it on other asyn tests I write

Reply:
It still gives the same warning after I install pytest-asyncio manually myself but im not sure what the issue actually is because google tells me putting asyncio_mode=auto in the config(which is already in there) would fix the issue.So appreciate it being included until I can find a solution.